src/core/nativecall.c

```diff
@@ -752,13 +752,13 @@ void MVM_nativecall_refresh(MVMThreadContext *tc, MVMObject *cthingy) {
         for (i = 0; i < repr_data->num_attributes; i++) {
             MVMint32 kind = repr_data->attribute_locations[i] & MVM_CSTRUCT_ATTR_MASK;
             MVMint32 slot = repr_data->attribute_locations[i] >> MVM_CSTRUCT_ATTR_SHIFT;
-            void *cptr;   /* The pointer in the C storage. */
-            void *objptr; /* The pointer in the object representing the C object. */
+            void *cptr = NULL;   /* The pointer in the C storage. */
+            void *objptr = NULL; /* The pointer in the object representing the C object. */
```

Comment:
These NULL initializations look like dead code (they are initialized on every code path).

Reply:
I know. It's just a habit I have with pointers. It's kinda safer if the code change bellow.

Reply:
Kinda, though one'd hope the compiler would catch that, and if it doesn't Valgrind or similar will.